recipes/muset/meta.yaml

```diff
@@ -0,0 +1,47 @@
+{% set name = "muset" %}
+{% set version = "0.5.1" %}
+
+package:
+  name: {{ name }}
+  version: {{ version }}
+
+source:
+  git_url: https://github.com/camiladuitama/muset.git
+  git_rev: v{{ version }}
+ 
+build:
+  number: 0
+  skip: True  # [not linux]
+  run_exports:
+    - {{ pin_subpackage(name, max_pin="x.x") }}
+
+requirements:
+  build:
+    - {{ compiler('cxx') }}
+    - cmake >=3.15
+    - make    
+  host:
+    - zlib
+  run:
+    - ggcat >=1.1.0
+
+test:
+  files:
+    - run_test.sh
+
+about:
+  home: "https://github.com/CamilaDuitama/muset"
+  license: GPL-3.0-or-later
```

Comment:
I've changed it from `MIT` to `GPL-3` because https://github.com/CamilaDuitama/muset/blob/main/LICENSE says GPL-3